graphql-ballerina/records.bal

```diff
@@ -101,17 +110,10 @@ public type __InputValue record {|
 	//string description?;
 |};
 
-public enum __TypeKind {
-    SCALAR,
-    OBJECT,
-    ENUM
-    // TODO: Support the following
-    //INTERFACE,
-    //UNION,
-    //INPUT_OBJECT,
-    //LIST,
-    //NON_NULL
-}
+
+
+# Represents the type kind of a GraphQL type.
+public type __TypeKind "SCALAR"|"OBJECT"|"ENUM"|"NON_NULL";
```

Comment:
This is changed since we cannot get the name of the `enum` type due to a limitation.